src/lib/helpers.test.ts

```diff
@@ -1162,3 +1163,23 @@ describe("validateHash", () => {
     expect(() => validateHash(validHash)).not.toThrow();
   });
 });
+
+describe("getSubstringAfterLastColon", () => {
+  test("returns the substring after the last colon", () => {
+    const input = "registry1.dso.mil/ironbank/opensource/defenseunicorns/pepr/controller:v0.31.1";
+    const output = getSubstringAfterLastColon(input);
+    expect(output).toEqual("v0.31.1");
+  });
+
+  test("returns empty string if there is no colon", () => {
+    const input = "registry1.dso.mil";
+    const output = getSubstringAfterLastColon(input);
+    expect(output).toBe("latest");
+  });
+
+  test("handles strings with multiple colons correctly", () => {
+    const input = "registry1:8080/dso.mil/pepr:latest";
+    const output = getSubstringAfterLastColon(input);
+    expect(output).toEqual("latest");
+  });
+});
```

Comment:
What happens if the input string is something like:

"registry1:8080/dso.mil/pepr"

Reply:
good catch, i added a unit test specifically around this one now